nacc_attribute_deriver/attributes/mqt/demographics.py

```diff
@@ -61,13 +61,22 @@ def _create_uds_primary_language(self) -> Optional[DateTaggedValue[str]]:
             if not primlang:
                 return None
 
+            primlang = int(primlang)
+
+            # if primlang is 9 and this is a followup packet, treat as None
+            # case. assumes the real value was set by an initial packet
+            if primlang == 9 and not self.__uds.is_initial():
```

Comment:
primlang isn't a variable in the FVP A1, so should just return None if the form isn't an initial visit.